static/style.css

```diff
@@ -104,8 +104,13 @@ body{
     justify-content: flex-end;
 }
 
+.HiddenMenu ul {
+    display: grid;
```

Comment:
I believe that a flex display is more interesting in this case, as the objective is to organize the items in a single direction, that is, in line. Grids are generally used when it is necessary to organize the contents in rows and columns.